src/components/TalentsListPage/components/SearchPanel/SearchPanel.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Input } from "../../../../shared/components";
+import s from "./SearchPanel.module.scss";
+
+export function SearchPanel() {
+    return (
+        <div className={s.search_panel}>
+            <Input id={s.long} className={s.input} type="text" placeholder="Name"/>
+            <Input className={s.input} type="text" placeholder="Specialization"/>
+            <Input className={s.input} type="text" placeholder="Skill"/>
+            <Button className={s.btn}>Search</Button>
```

Comment:
why do u use tags with capitals? u just can add classNames for them for facilitation of understanding

Reply:
this small inconvenience allows us to get rid of code copying